crates/node/engine/src/lib.rs

```diff
@@ -6,6 +6,10 @@
 )]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
+#[cfg(feature = "metrics")]
+#[macro_use]
+extern crate kona_macros;
```

Comment:
Nice!

Reply:
Hi, if this looks good, can I pick up the other metrics that need to be added here?